commitlint.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
     extends: ['@commitlint/config-conventional'],
     rules: {
-        "scope-enum": [2, "always", ["deps", "configurations"]],
+        "scope-enum": [2, "always", ["deps", "configurations", "api", "convert"]],
```

Comment:
`api` is not a good scope

if you accept that we need to change the API to be by `source/{source_name}` then a good scope would be `external-sources`

Reply:
Done